shell/platform/embedder/embedder.h

```diff
@@ -306,12 +306,26 @@ typedef bool (*TextureFrameCallback)(void* /* user data */,
                                      FlutterOpenGLTexture* /* texture out */);
 typedef void (*VsyncCallback)(void* /* user data */, intptr_t /* baton */);
 
+typedef struct {
+  /// The width of the surface that will be backed by the fbo.
+  uint32_t width;
+  /// The height of the surface that will be backed by the fbo.
+  uint32_t height;
+} FlutterFrameInfo;
+
+typedef uint32_t (*FBOWithFrameInfoCallBack)(void* /* user data */,
```

Comment:
I don't believe prepositions are used in in the embedder API. Also, the `uint32_t` needing to be an FBO is not really required. How about `UIntFrameInfoCallback` instead to keep this consistent with the `UIntCallback` you are upgrading?

Reply:
Done